### DISCUSSION TITLE
docker "-headless" option fail

### DISCUSSION THREAD

**Discussion by leveryd** [projectdiscovery/katana#259] [Q&A] [answered]
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/1846319/210336730-5528fdcc-6aa6-4438-93fc-c9b4f5dc89d0.png">

**Comment by ehsandeep** [comment, marked as answer]
@leveryd see https://github.com/projectdiscovery/katana/issues/154#issuecomment-1309177635, use `-sc` option
